app/forms/access-util.tsx

```diff
@@ -55,3 +76,58 @@ export type EditRoleModalProps = AddRoleModalProps & {
   identityType: IdentityType
   defaultValues: { roleName: RoleKey }
 }
+
+type RoleRadioFieldProps = {
+  control: Control<AddUserValues> | Control<{ roleName: RoleKey }>
+  scope: 'Silo' | 'Project'
+}
+
+export function RoleRadioField({ control, scope }: RoleRadioFieldProps) {
+  const roleDescriptions = scope === 'Silo' ? siloRoleDescriptions : projectRoleDescriptions
+  const currentRole = control._formValues.roleName || ''
```

Comment:
Shouldn't do this, should be something like:

```ts
const currentRole = useWatch({ control, name: 'roleName' })
```

Reply:
6e0af828b24528b61121ba7aa181cc0a7ae188c1 fixes this a different way: this only needed to be here because the mechanism inside `RadioFieldDyn` that checks the right radio based on the field value was getting messed up by a `div` being used to put space between the radios. I got rid of the div and fixed the field state thing, but now we need to figure out how to put the spacing back. The `className` prop on `Radio` is unfortunately not it because it doesn't get stuck on the outermost div. We could just add another prop for it, something silly like `containerClass`.